lib/tasks/upsert_vita_partners.rake

```diff
@@ -1,32 +1,9 @@
-## these tasks manage VITA partner information
-require 'yaml'
-
+##
+# these tasks manage VITA partner information
 namespace :db do
   desc 'loads new vita partners, updates existing partners'
   task upsert_vita_partners: [:environment] do
-    puts "beginning partner upsert using environment: #{Rails.env}"
-    # TODO: don't leave the next line in
-    # load yaml file
-    partners = YAML.load_file('db/vita_partners.yml')['vita_partners']
-    partners.each do |datum|
-      partner = VitaPartner.find_by(zendesk_group_id: datum['zendesk_group_id'])
-      partner.present? ? update_partner(partner, datum) : create_partner(datum)
-    end
+    include VitaPartnerImporter
+    upsert_vita_partners
```

Comment:
Is there a reason to not make these classes (so you could just call `VitaPartnerImporter.upsert_vita_partners`)? I think mixing in the methods makes me a bit less comfortable that I understand the full effect of what methods are defined in which scopes.

Reply:
There is! Since the class doesn't encapsulate any information, there's no need for the class overhead.